src/mantarray_desktop_app/main.py

```diff
@@ -127,6 +127,13 @@ def _log_system_info() -> None:
         logger.info(msg)
 
 
+class Filter_System_Status_Logs(logging.Filter):
+    def filter(self: Filter_System_Status_Logs, record: logging.LogRecord) -> bool:
+        # log system_status only if not 200
+        message = record.getMessage()
+        return not ("system_status" in message and " 200 " in message)
```

Comment:
What happens if 200 is in this log line but it's not the return code?

Reply:
Also remember to add tests. One of them should test this edge case

Reply:
Initially I added spaces between 200 " 200 " to look for a 200 status code but I changed it to look in a specific location for the error code and check if it is 200 or not.